mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -634,3 +634,58 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += '(num_points={})'.format(self.num_points)
         return repr_str
+
+
+@PIPELINES.register_module()
+class BackgroundPointsFilter(object):
+    """Filter background points near the bounding box.
+
+    Args:
+        bbox_enlarge_range (list[float]): Point cloud range.
+    """
+
+    def __init__(self, bbox_enlarge_range):
+        self.bbox_enlarge_range = np.array(
```

Comment:
1. use tuple rather than list and assert mmcv.is_tuple_of(bbox_enlarge_range, int)
2. Could bbox_enlarge_range be a single float number?